openoa/types/plant.py

```diff
@@ -10,7 +10,7 @@
 from openoa.types import timeseries_table
 
 from .asset import AssetData
-from .reanalysis import ReanalysisData
+from ..utils.reanalysis import ReanalysisData
```

Comment:
Let's remove these relative imports.

Reply:
This is resolved here: https://github.com/NREL/OpenOA/pull/198/commits/e92782a485a0475d68cc60e36c126662a41181a0